release/ansible/Dockerfile.rhel8

```diff
@@ -20,18 +21,20 @@ ENV HOME=/opt/ansible \
     USER_NAME=ansible \
     USER_UID=1001
 
+# get artifacts from cachito
+COPY $REMOTE_SOURCE/cachito-gomod-with-deps/app $REMOTE_SOURCE_DIR
+WORKDIR $REMOTE_SOURCE_DIR/app
+
 RUN yum install -y \
       ansible \
       ansible-runner \
       ansible-runner-http \
```

Comment:
I *think* these are also python packages that we could install with pip. For reference, this is the upstream Pipfile that matches with the packages in this dockerfile: https://github.com/operator-framework/operator-sdk/blob/9d2f672c8e5bb0af8d0b5955cfa673aad21f1281/images/ansible-operator/Pipfile

Reply:
Yea I was avoiding changing too many things, since I'd expect downstream for these packages to be installed from RPMs provided in the appropriate content_set. We could look at changing this in the future, but for now, I wanted to limit the changes to just the self-maintained RPM packages.